T.A.Random Colors/index.js

```diff
@@ -0,0 +1,24 @@
+const btn = document.querySelector('button');
+const body = document.getElementsByTagName('BODY');
+
+btn.addEventListener('click',function(e) {
+    e.preventDefault();
+    makeColor();
+    changeColor();
+});
+
+function changeColor() {
+    document.body.style.background = makeColor();
+}
+
+function makeColor() {
+    const arr = [];
```

Comment:
Explain to me what this function does.

Reply:
this is the empty array and I loop over the random color and push that empty array